dotCMS/src/main/java/com/dotcms/rest/IntegrityResource.java

```diff
@@ -230,79 +257,139 @@ public Response generateIntegrityData(@Context HttpServletRequest request, @Form
     @Path("/getintegritydata/{params:.*}")
     @Produces("application/zip")
     @Consumes(MediaType.MULTIPART_FORM_DATA)
-    public Response getIntegrityData(@Context HttpServletRequest request, @FormDataParam("AUTH_TOKEN") String auth_token_digest, @FormDataParam("REQUEST_ID") String requestId)  {
-        String remoteIP = null;
+    public Response getIntegrityData(@Context final HttpServletRequest request,
+                                     @FormDataParam("AUTH_TOKEN") final String auth_token_digest,
+                                     @FormDataParam("REQUEST_ID") final String requestId)  {
```

Comment:
Codacy found an issue: [The String literal "REQUEST_ID" appears 4 times in this file; the first occurrence is on line 225](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6151243)